config/default.js

```diff
@@ -21,63 +20,49 @@ config.app = {
  * Configure network
  */
 
-var bitcoinNetwork = process.env.BITCOIN_NETWORK
-
-if (bitcoinNetwork) {
-  bitcore.Networks.defaultNetwork = bitcore.Networks[bitcoinNetwork]
-} else {
-  bitcore.Networks.defaultNetwork = bitcore.Networks.testnet
-}
-
-config.currencies = defer(function () {
-  let bitcoinNetworks = {}
-
-  bitcoinNetworks[bitcoinNetwork] = {
-    incomingPrivateKey: process.env.BITCOIN_HD_PRIVATE_KEY,
-    outgoingPublicKey: process.env.BITCOIN_HD_PUBLIC_KEY,
-    documentPrice: parseInt(process.env.DOCUMENT_PRICE),
-    feeMultiplier: parseFloat(process.env.FEE_MULTIPLIER || 2)
-  }
-
-  return {
-    btc: {
-      defaultNetwork: bitcoinNetwork,
-      networks: bitcoinNetworks
-    }
-  }
+config.networkName = defer(function () {
+  return this.app.defaultNetwork === 'testnet' ? 'test3' : 'main'
 })
 
 /**
  * Configure prices.
  */
 
 config.documentPrice = defer(function () {
-  let btc = this.currencies.btc
-  let documentPrice = btc.networks[btc.defaultNetwork].documentPrice
+  const defaultChain = this.app.defaultChain
+  const defaultNetwork = this.app.defaultNetwork
+
+  let chain = this.chains[defaultChain]
+  let documentPrice = chain.networks[defaultNetwork].documentPrice
   return documentPrice
 })
 
 config.feeMultiplier = defer(function () {
-  let btc = this.currencies.btc
-  let documentPrice = btc.networks[btc.defaultNetwork].feeMultiplier
-  return documentPrice
+  const defaultChain = this.app.defaultChain
+  const defaultNetwork = this.app.defaultNetwork
+
+  let chain = this.chains[defaultChain]
+  let feeMultiplier = chain.networks[defaultNetwork].feeMultiplier
+  return feeMultiplier
 })
 
 /**
  * Configure Insight.
  */
 
 config.insightUrl = defer(function () {
-  const btc = this.currencies.btc
+  const defaultNetwork = this.app.defaultNetwork
+
   const insight = this.services.insight
-  const url = insight[btc.defaultNetwork].url
+  const url = insight[defaultNetwork].url
   return url
 })
 
 config.insightApiUrl = defer(function () {
-  const btc = this.currencies.btc
+  const defaultNetwork = this.app.defaultNetwork
```

Comment:
a detail, but in ES6 you can unpack values like `const { defaultNetwork }  = this.app`

Reply:
Not much point for one variable. :)